docs/deployment/forked_threaded_servers.rst

```diff
@@ -0,0 +1,94 @@
+Forked and Threaded Servers
++++++++++++++++++++++++++++
+
+Forked and Threaded Servers share common "gotchas" and solutions when
+using pyramid and some popular packages.
+
+Forked and Threaded servers tend to use a "copy on write" implementation detail
+to optimize how they work and share memory. This can create problems when
+certain actions happen before the fork or thread dispatch, such as when files or
+file-descriptors are opened or random number generators are initialized.
+
+Many servers have built-in hooks or events which allow you to easily handle these
+situations.
+
+
+Servers
+++++++++++
+
+gunicorn
+========
+
+gunicorn offers several hooks during an application lifecycle
+
+the postfork routine is provided as a function in a configuration python script.
+
+for example a script 'config.py' might look like:
+
+	def post_fork(server, worker):
+		log.debug("gunicorn - post_fork")
+
+and invoking the script would look like:
+
+	gunicorn --paste production.ini -c config.py
+
+documentation for the `post_fork` hook appears on http://docs.gunicorn.org/en/latest/settings.html#post-fork
+
+
+uwsgi
+========
+
+uwsgi offers a decorator to handle forking
+
+your application would include code like
+
+    from uwsgidecorators import postfork
+    
+    @postfork
+    def my_setup():
+		log.debug("uwsgi - postfork")
+
+documentation for the `postfork` decorator appears on https://uwsgi-docs.readthedocs.io/en/latest/PythonDecorators.html#uwsgidecorators.postfork
+
+
+waitress
+========
+
+waitress is not a forking server, but it's threads can create similar issues as forking servers
```

Comment:
Capitalization, "its", period.

Waitress is not a forking server, but its threads can create similar issues as forking servers.